src/main/resources/static/scripts.js

```diff
@@ -9,16 +9,71 @@ function lightModeToggle() {
     }
 }
 
+function openModal() {
+    const modal = document.getElementById("myModal");
+
+    modal.style.display = "block";
+}
+
+function resetModal() {
+    const modalContent = document.getElementById("modal-content");
+    let closeSpan = document.createElement("span");
+    closeSpan.setAttribute("class", "close");
+    closeSpan.innerText = '×';
+
+    modalContent.innerHTML = '';
+    modalContent.appendChild(closeSpan);
+    modalWindow();
+}
+
+function viewKeybindsModal() {
+    const modalContent = document.getElementById("modal-content");
+
+    let pElement = document.createElement("p");
+    let text = document.createTextNode("Implement Thymeleaf in here to pull existing keybinds");
+
+    resetModal()
+    pElement.appendChild(text);
+    modalContent.appendChild(pElement);
+    openModal();
+}
+
+function helpModal() {
```

Comment:
This function adds the help pop up/modal. I apologize for it not being particularly useful as it is and including placeholder text. I do not know enough about the app or using Lua to define keybinds in general to make a real guide here. However, the basic modal functionality is included. The content is reset and sections are added, then the modal is opened.

Reply:
Somehow snipped the wrong set of lines but I'm talking about the function called helpModal() starting from line 41